fhir-config/src/main/java/com/ibm/fhir/config/FHIRConfigHelper.java

```diff
@@ -151,5 +155,40 @@ private static JsonValue getPropertyFromTenantOrDefault(String propertyName) {
         }
 
         return (result != null ? result : defaultValue);
+    }    
+    
+    /**
+     * This method returns the list of supported resource types
+     * @return a list of resource types that isn't null
+     */
+    public static List<String> getSupportedResourceTypes() throws FHIRException {
+        List<String> result = new ArrayList<>();
+
+        PropertyGroup rsrcsGroup = FHIRConfigHelper.getPropertyGroup(FHIRConfiguration.PROPERTY_RESOURCES);
+        List<PropertyEntry> rsrcsEntries;
+        try {
+            rsrcsEntries = rsrcsGroup.getProperties();
+
+            if (rsrcsEntries != null && !rsrcsEntries.isEmpty()) {
+                for (PropertyEntry rsrcsEntry : rsrcsEntries) {
+                    String name = rsrcsEntry.getName();
```

Comment:
not always flawless... maybe add a check to see if it's actually a correct/proper resource name?

Reply:
Paul and I talked and it's not possible to check this at this level of the code.  I added a comment stating that.